frontend/public/components/monitoring/metrics.tsx

```diff
@@ -453,7 +453,13 @@ export const QueryBrowserPage = withFallback(() => {
 
   let colorOffset = 0;
 
-  const queryStrings = _.map(queries, 'query');
+  const queriesMemoKey = JSON.stringify(_.map(queries, 'query'));
```

Comment:
We are doing the same (maybe even a more expensive) calculation here to create a memo key. We might as well just take the `React.useMemo` statements out and calculate the values on every render, since we are doing the same operation to create a key.

Reply:
Sorry, I should have explained better. I'm using `React.useMemo` to prevent re-renders rather than to save time calculating these values. They are passed as props to child components and if they are recalculated on every render, they will always be new arrays, causing child components to re-render. Normally, this wouldn't be an issue, but in this case the graph is a child component and is slow to re-render.

Reply:
Hmm, okay. That makes sense. Could we add a comment somewhere in here to explain that?

Reply:
👍 Added